### ISSUE TITLE
plot_model error: Error in pretty.default(c(floor(lower_lim), ceiling(upper_lim)))...

### ISSUE THREAD

**Issue by wthogmartin** [strengejacke/sjPlot#846]
I believe the issue is similar to https://github.com/strengejacke/sjPlot/issues/845 thought the error message differs

The error message in question: 
Error in pretty.default(c(floor(lower_lim), ceiling(upper_lim))) : 
  7 arguments passed to .Internal(pretty) which requires 8

A reproducible example:
```r
dt<-data.frame(y=rnorm(100),x=rnorm(100),z=rnorm(100), A=sample(c('group 1', 'group 2', 'group 3'), 100, replace=TRUE))
m0 <- lm(y ~ A*x + A*z, data=dt)
sjPlot::plot_model(m0)
sjPlot::plot_model(m0, type = "eff")
sjPlot::plot_model(m0, type = "pred")

m1 <- lm(y ~ x + z, data=dt)
sjPlot::plot_model(m1)

m2 <- lm(y~x, data=dt)
sjPlot::plot_model(m2)
```

all return the same error.

I've tried various 8 argument plots, but identifying which argument is missing has been unsuccessful so far

**Comment by strengejacke** [comment]
Cannot reproduce, works fine for me:

``` r
dt <-
  data.frame(
    y = rnorm(100),
    x = rnorm(100),
    z = rnorm(100),
    A = sample(c('group 1', 'group 2', 'group 3'), 100, replace = TRUE)
  )
m0 <- lm(y ~ A * x + A * z, data = dt)
sjPlot::plot_model(m0)
```

![](https://i.imgur.com/J2OMXh7.png)

``` r
sjPlot::plot_model(m0, type = "eff")
#> $A
```

![](https://i.imgur.com/wrijwT4.png)

    #> 
    #> $x

![](https://i.imgur.com/b3REWZZ.png)

    #> 
    #> $z

![](https://i.imgur.com/9qJheaZ.png)

``` r
sjPlot::plot_model(m0, type = "pred")
#> $A
```

![](https://i.imgur.com/5U6qIQs.png)

    #> 
    #> $x

![](https://i.imgur.com/Pf1LWRo.png)

    #> 
    #> $z

![](https://i.imgur.com/fotyQvt.png)

``` r
m1 <- lm(y ~ x + z, data = dt)
sjPlot::plot_model(m1)
```

![](https://i.imgur.com/WEoUadk.png)

``` r
m2 <- lm(y ~ x, data = dt)
sjPlot::plot_model(m2)
```

![](https://i.imgur.com/3Nzj38s.png)

<sup>Created on 2022-07-31 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by wthogmartin** [comment]
I should have indicated this earlier but I am working with R 4.2 and sjPlot 2.8.10.

**Comment by wthogmartin** [comment]
The problem was resolved updating from R 4.2 to R 4.2.1

**Comment by wthogmartin** [comment]
The problem was resolved updating from R 4.2 to R 4.2.1
